hack/jenkins/job-configs/kubernetes-e2e-trusty.yaml

```diff
@@ -0,0 +1,75 @@
+- job-template:
+    name: 'kubernetes-e2e-gce-trusty-{suffix}'
+    description: '{description} Test owner: {test-owner}.'
+    logrotate:
+        daysToKeep: 7
+    builders:
+        - shell: |
+            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e.sh" | bash -
+    properties:
+        - mail-watcher
+    publishers:
+        - claim-build
+        - junit-publisher
+        - log-parser
+        - email-ext:
+            recipients: "{emails}"
+        - description-setter:
+            regexp: KUBE_GCE_MINION_IMAGE=(.*)
+        - groovy-postbuild: |
```

Comment:
This is giving me a warning. I think you're supposed to use the `script` parameter as documented [here](http://docs.openstack.org/infra/jenkins-job-builder/publishers.html#publishers.groovy-postbuild).

`WARNING:jenkins_jobs.modules.publishers:groovy-postbuild:You use depricated configuration, please follow documentation to change configuration. It is not going to be supported in future releases!`